clang/test/SemaSYCL/prohibit-thread-local.cpp

```diff
@@ -0,0 +1,49 @@
+// RUN: %clang_cc1 -fsycl-is-device -verify -fsyntax-only -std=c++17 %s
+
+thread_local const int prohobit_ns_scope = 0;
+thread_local int prohobit_ns_scope2 = 0;
+thread_local const int allow_ns_scope = 0;
```

Comment:
```suggestion
thread_local const int allow_ns_scope = 0;
```
Were you planning to use this somehow?

Reply:
I don't see the change being suggested?  

I intended to not use it to show that we wouldn't diagnose there.